pkg/skaffold/build/local/local.go

```diff
@@ -35,6 +35,11 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// For testing
+var (
```

Comment:
I don't really understand this specific part? Does this just become a variable in the scope of the local pkg? Was there some problem with using the direct reference (`docker.RemoteDigest`) in the jib files?

Reply:
This is the simplest way to mock the call to `docker.RemoteDigest`. We use that pattern in. a lot of places.